src/components/LinkBox/LinkBox.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { IconDefinition } from '@fortawesome/fontawesome-common-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import './style.css';
+
+import {
+  LinkBoxText,
+  LinkBoxTitle,
+  LinkBoxWrapper,
+  TextWrapper,
+} from './styled';
+
+type LinkBoxProps = {
+  title: string;
+  text: string;
+  icon: IconDefinition;
+  navigate: string;
+};
+
+function LinkBox({ title, text, icon, navigate }: LinkBoxProps) {
+  return (
+    <LinkBoxWrapper>
```

Comment:
You forgot to add the navigate prop to the LinkBoxWrapper

```suggestion
     <LinkBoxWrapper href={navigate}>
```